rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/370_profile.yml

```diff
@@ -163,7 +169,8 @@ disabling stored fields removes fetch sub phases:
 
   - match: { hits.hits.0._index: test }
   - match: { profile.shards.0.fetch.debug.stored_fields: [] }
-  - is_false: profile.shards.0.fetch.children
+  - length: { profile.shards.0.fetch.children: 1 }
+  - match: { profile.shards.0.fetch.children.0.type: FetchFieldsPhase }
```

Comment:
That was the purpose of the `fetchStoredFields` flag.

Reply:
That would break another test in `370_profile` when checking

```
- match: { profile.shards.0.fetch.debug.stored_fields: [_id, _routing, _source] }
```